WeScanSampleProject/EditImageViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  EditImageViewController.swift
+//  WeScanSampleProject
+//
+//  Created by Chawatvish Worrapoj on 8/1/2020
+//  Copyright © 2020 WeTransfer. All rights reserved.
+//
+
+import UIKit
+import WeScan
+
+final class EditImageViewController: UIViewController {
```

Comment:
Why do we need to extend the `EditImageViewController` like this in the sample project?

Reply:
I think I didn't extend anything from EditImageViewController. In this sample project, I just create an empty view controller and add child EditImageViewController into view block.

Anyway, maybe it's my fail because I use same name of view controller name.